core/src/main/java/com/graphhopper/routing/Path.java

```diff
@@ -377,7 +377,7 @@ public InstructionList calcInstructions(final Translation tr) {
      * @return List of PathDetails for this Path
      */
     public Map<String, List<PathDetail>> calcDetails(final PathDetailsBuilderFactory pathBuilderFactory, int previousIndex) {
-        if (pathBuilderFactory == null)
+        if (!isFound() || edgeIds.isEmpty() || pathBuilderFactory == null)
```

Comment:
Hmmh, we should add an integration test and not calling PathMerger at all or wdyt?